package.json

```diff
@@ -38,67 +25,37 @@
     "@angular/platform-browser": "4.1.2",
     "@angular/platform-browser-dynamic": "4.1.2",
     "@angular/router": "4.1.2",
-    "@ngx-translate/core": "^6.0.1",
-    "@ngx-translate/http-loader": "0.0.3",
+    "@ngx-translate/core": "^7.0.0",
+    "@ngx-translate/http-loader": "^0.1.0",
     "core-js": "^2.4.1",
-    "moment": "2.18.1",
-    "ngx-clipboard": "5.0.1",
-    "reflect-metadata": "^0.1.3",
-    "rxjs": "5.4.0",
-    "uuid": "^3.0.1",
-    "zone.js": "0.8.10"
+    "moment": "^2.18.1",
```

Comment:
Importing moment also includes all locales. The workaround is to use context replacement plugin in webpack [https://github.com/moment/moment/issues/2517#issuecomment-185836313](url). Need to find a way to do this since angular cli doesn't expose the config